scipy/signal/filter_design.py

```diff
@@ -1033,6 +1033,33 @@ def zpk2sos(z, p, k, pairing='nearest'):
     return sos
 
 
+def _align_nums(nums):
+    """
+    Given an array of numerator coefficient arrays [[a_1, a_2,...,
+    a_n],..., [b_1, b_2,..., b_m]], this function pads shorter numerator
+    arrays with zero's so that all numerators have the same length. Such
+    alignment is necessary for functions like 'tf2ss', which needs the
+    alignment when dealing with SIMO transfer functions.
+    """
+    try:
+        nums = asarray(nums)
+
+        if not np.issubdtype(nums.dtype, np.number):
+            raise ValueError("dtype of numerator is non-numeric")
+
+        return nums
+
+    except ValueError:
```

Comment:
Since that wasn't clear, I will add a comment about that just so people will know for future reference.